src/jsTyping/jsTyping.ts

```diff
@@ -31,7 +31,6 @@ import {
     versionMajorMinor,
 } from "./_namespaces/ts";
 
-/** @internal */

```

Comment:
Instead of removing internal can we re-export it in server as some other name so that typescript.d.ts doesnt change.